test_data/sample-app.yaml

```diff
@@ -15,6 +15,10 @@ spec:
       labels:
         app: myapp
     spec:
+      initContainers:
```

Comment:
Can you please regenerate chart examples with `cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app` and push changes to your branch?

Reply:
Hey Artem, the examples are updated per your request. Thanks for the quick reply.